### PR TITLE
[hw,rv_dm,dv] Enable CDC instrumentation

### PR DESCRIPTION
**Pull request by crteja**
- Enable CDC instrumentation for `RV_DM`
- Override timeout of `rv_dm_csr_aliasing` to fix timeout error

This PR partially addresses https://github.com/lowRISC/opentitan/issues/16689

### CODE REVIEW

File: hw/ip/rv_dm/dv/rv_dm_sim_cfg.hjson
Comment (marnovandermaas) [inline hw/ip/rv_dm/dv/rv_dm_sim_cfg.hjson:84]: Why do you decrease the reseed by so much? In the life cycle controller they reseed 10 times.

**Review by marnovandermaas — COMMENTED** [review]
Some initial comments.

File: hw/ip/rv_dm/dv/rv_dm_sim_cfg.hjson
Comment (crteja) [reply hw/ip/rv_dm/dv/rv_dm_sim_cfg.hjson:80]: yes, this is from the `csr_tests`. I have copied to override the timeout paramater
https://github.com/lowRISC/opentitan/blob/c6d13942ec3df7be63ff6ab172f59823d3e9be3c/hw/dv/tools/dvsim/tests/csr_tests.hjson#L44-L50

File: hw/ip/rv_dm/dv/rv_dm_sim_cfg.hjson
Comment (crteja) [reply hw/ip/rv_dm/dv/rv_dm_sim_cfg.hjson:84]: This is the default from `csr_tests`. I have not changed it.

**Review by marnovandermaas — APPROVED** [review]
Thanks for answering my questions. Looks good to me.

**Comment by marnovandermaas** [comment]
Let's wait for CI to pass before merging.

**Pull request merged**
